scripts/data_overlap/get_group_overlap_stats.py

```diff
@@ -0,0 +1,91 @@
+import json
+import cattrs
+import os
+import argparse
+from typing import List, Dict
+
+from common.general import asdict_without_nones
+
+from data_overlap_spec import DataOverlapStats
+from light_scenario import GroupScenarioSpecs, GroupOverlapStats, AllGroupOverlapStats
+
+
+def save_group_overlap_stats_to_jsonl(all_group_overlap_stats: AllGroupOverlapStats, filename: str):
+    with open(filename, "a") as f:
+        f.write(json.dumps(asdict_without_nones(all_group_overlap_stats), ensure_ascii=False) + "\n")
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
```

Comment:
In general, could add more documentation on what this file is trying to do

Reply:
Added